components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityConfigParser.java

```diff
@@ -374,6 +386,60 @@ private void buildCookieConfig()    {
         }
     }
 
+    public void buildLegacyFeatureConfig() {
```

Comment:
In https://github.com/GANGANI/carbon-identity-framework/blob/76ad8e0307b509aefc1c7e7716756ad143850105/components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityUtil.java#L228-L236, this Map value was used. Since some configs do not have a feature version, adding a separator would not be appropriate when retrieving the value from the Map. WDYT?

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/3109/commits/0b1c211bf21195cc5e917d052c9fda0f51a7e800